README.md

```diff
@@ -1,32 +1,116 @@
-# Background
+# Supplementary Data Service (sds)
 
-This is the repo for the Supplementary Data Service (SDS), which is described in 
-https://confluence.ons.gov.uk/display/SDC/SDS
+More information on this service can be found on Confluence:
 
-# Running locally
+* https://confluence.ons.gov.uk/display/SDC/SDS
+
+## Running locally
 
 To run this service locally, you will need the following:
 
 * Python 3.11
-* A Google Firebase key file (see https://www.youtube.com/watch?v=MU7O6emzAc0)
-* Docker or Rancher desktop (optional but needed if you want to run the docker image)
+* VirtualEnv
+* Docker
 
-To run, set up a local virtual environment and install the libraries described in the requirements file (see below).
+### Creating, activating and deactivating a Python virtual environment
 
-```bash
-python venv .venv
-. .venv/bin/activate
+Check that you have the correct version of Python installed and then run the following commands:
+
+```
+virtualenv venv
+source venv/bin/activate
+```
+
+This will create and activate the virtual environment. To deactivate the environment, run the following command:
+
+```
+deactivate
+```
+
+You can use another name for the virtual environment's directory if you wish but please add it to the `.gitignore` file.
+
+### Installing the dependencies
+
+Assuming that the virtual environment is activated, run the following command:
+
+```
 pip install -r requirement.txt
 ```
 
```

Comment:
Worth adding in either here or after setting up environment variables, the command to run the application in the virtual environment? "uvicorn src.app:app --reload"